packages/sit-onyx/src/components/OnyxStepper/types.ts

```diff
@@ -33,18 +33,32 @@ export type OnyxStepperProps = FormInjectedProps &
     step?: number; // step-mismatch + step-increment
 
     /**
-     * The smallest allowed value and rounded precision
+     * Number of decimal places to show. Can also be negative. Value will be rounded if needed
+     * to match the specified precision.
+     *
+     * @example For `precision=2` with `modelValue=5`, "5.00" will be displayed.
+     * @example For `precision=-1` with `modelValue=60`, "100" will be displayed.
```

Comment:
Just tested this, when I enter 60 -> 60,0 is displayed. Am I missing something or the description is wrong?

Reply:
sorry, the description should be -2 instead of -1. I changed it.